raiden/ui/cli.py

```diff
@@ -746,6 +757,26 @@ def run(ctx, **kwargs):
     from raiden.ui.console import Console
     from raiden.api.python import RaidenAPI
 
+    if kwargs['config_file']:
+        try:
+            config_file = toml.load(kwargs['config_file'])
+            addr_options = {'address', 'registry_contract_address', 'discovery_contract_address'}
```

Comment:
Please generate this set dynamically by inspecting the params `type` (see below for the example on how to get the default values). Address parameters will have a type of `raiden.utils.cli.AddressType`.